ui/app/config.js

```diff
@@ -99,6 +110,19 @@ ConfigScreen.prototype.render = function () {
   )
 }
 
+function rpcValidation (newRpc, state) {
+  if (validUrl.isWebUri(newRpc)) {
+    state.dispatch(actions.setRpcTarget(newRpc))
+  } else {
+    var appendedRpc = `http://${newRpc}`
+    if (validUrl.isWebUri(appendedRpc)) {
+      state.dispatch(actions.displayWarning('URIs require the appropriate HTTP/HTTPS prefix.'))
```

Comment:
I actually don't think we should bother trying any prefix, since we don't know what protocol the actual address should refer to.  Showing a warning should be sufficient.